test/integration/test_noise_learner.py

```diff
@@ -33,7 +34,10 @@ class TestIntegrationNoiseLearner(IBMIntegrationTestCase):
 
     def setUp(self) -> None:
         super().setUp()
-        self.backend = "test_eagle"
+        try:
+            self.backend = self.service.backend("test_eagle")
+        except:
+            raise SkipTest("test_eagle not available in this environment")
```

Comment:
Going to skip tests if test_eagle is not available, we can update this as a part of https://github.com/Qiskit/qiskit-ibm-runtime/pull/1828

I'll also need to reach out and make sure the test accounts we use all have access to `test_eagle` and `test_eagle_us-east`